### DISCUSSION TITLE
Simplified Interaction Mode Enquiry

### DISCUSSION THREAD

**Discussion by qiangjiadong** [mozilla/hubs#3552] [Hubs Cloud Questions] [answered]
Hi guys, I’ve been customising my Hubs Cloud for Museum and Art Institution.

During a series of discussions and feedback, I just wanna ask if it is possible to customise a **simplified version of interaction** mode, particularly when users enter the hubs room.

To be honest, currently both of desktop version and mobile require TOO MUCH steps of choosing VR/web view, enable mic, choosing avatar, etc. It does makes lots of users without knowledge of game and 3D space very confused.

I found inside the **lobby version, and clicking the button of Hide All** is a VERY GOOD and SIMPLE mode of presentation of 3D space, it allows various users to start to explore without any worries on mic or avatar etc.

May I ask, is there a way to make this mode (**Lobby Version plus Hide All) as Default** when users just enter the room. Once they are interested, they might be able to click the sandwich button and try to further interact.

I would love to hear alternative solutions to achieve a better simplified mode similar to my ideas above.
Thks a lot!!!!

**Comment by robinkwilson** [comment, marked as answer]
Hi @qiangjiadong !

That change would require a custom client and editing the code to set those buttons as not seen by default: https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

That being said, we're in the process of working on a REALLY EXCITING UI REDESIGN: https://github.com/mozilla/hubs/pull/3006 (you're not alone in thinking the UI is a bit cluttered :) )

So the UI will be a LOT different in the future - no planned release date yet, we're squashing bugs at the moment. Hope this helps!

**Reply by robinkwilson** [reply]
Marking as answered, please "unmark" and add more information if we need to revisit.

**Reply by qiangjiadong** [reply]
Thks Robin!
